db/migrate/20190325064225_create_accepted_comment_likes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateAcceptedCommentLikes < ActiveRecord::Migration[5.2]
+  def change
+    def up
+      answers = Answer.where(accepted: true)
+      timestamp_to_answer_id = answers.group('created_at').minimum('id')
+                                         .inject({}) {|hash, (key, value)| hash[key.to_i] = value; hash}
```

Comment:
Oh, is this supposed to be `inject {|hash, (key, value)| ...` ? Sorry, a little hard to read...

Reply:
yes, hash is the new injected hash and key, value gives corresponding keys and values at each iteration